package.json

```diff
@@ -24,14 +24,18 @@
     "prepublish": "npm run build"
   },
   "jest": {
-    "testPathIgnorePatterns": ["/example"]
+    "testPathIgnorePatterns": [
+      "/example"
+    ]
   },
   "license": "MIT",
   "dependencies": {
-    "@cycle/xstream-run": "^4.0.0",
-    "xstream": "^9.0.0"
+    "@cycle/run": "*",
+    "xstream": "10.2"
   },
   "devDependencies": {
+    "@cycle/rxjs-run": "4.0.0-rc.5",
```

Comment:
Do you think it could make sense to initially just extract the drivers into the userland, and leave Unified for later, e.g. when it's released?